<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_CadToolBarPolylines</class>
 <widget class="QWidget" name="QG_CadToolBarPolylines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>56</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>56</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Polylines</string>
  </property>
  <widget class="QToolButton" name="bBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Back to main menu</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/back.png</normaloff>:/extui/back.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolyline">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create Polyline</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polyline.png</normaloff>:/extui/polyline.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolylineTrim">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Trim segments</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polylinetrim.png</normaloff>:/extui/polylinetrim.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolylineDelBetween">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>76</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete between two nodes</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polylinedelbetween.png</normaloff>:/extui/polylinedelbetween.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolylineDel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete node</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polylinedel.png</normaloff>:/extui/polylinedel.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolylineAdd">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>20</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add node</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polylineadd.png</normaloff>:/extui/polylineadd.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="bPolylineAppend">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Append node</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../res/extui/extui.qrc">
     <normaloff>:/extui/polylineappend.png</normaloff>:/extui/polylineappend.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">qg_actionhandler.h</include>
 </includes>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bPolyline</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>drawPolyline()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBack</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPolylineAdd</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>polylineAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPolylineDel</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>polylineDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPolylineDelBetween</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>polylineDelBetween()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPolylineTrim</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>polylineTrim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPolylineAppend</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarPolylines</receiver>
   <slot>polylineAppend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
